blocks/init/src/Blocks/custom/column/column-editor.scss

```diff
@@ -1,10 +1,15 @@
-@import './styles/shared';
+.block-column {
+	display: flex;
+	width: 100%;
+	opacity: var(--column-opacity, 1);
+	transform: var(--column-transform);
 
-.block-column.wp-block.block-editor-block-list__block.block-column {
-	margin-left: 0;
-	margin-right: 0;
-
-	@include column;
+	transition: {
+		// stylelint-disable-next-line declaration-no-important
+		property: opacity, transform !important;
```

Comment:
how come you use important?

Reply:
Have to override `opacity` from some default Gutenberg style.